mleap-spark/src/test/scala/org/apache/spark/ml/classification/parity/LinearSVCParitySpec.scala

```diff
@@ -0,0 +1,30 @@
+package org.apache.spark.ml.classification.parity
+
+import org.apache.spark.ml.classification.LinearSVCModel
+import org.apache.spark.ml.feature.{StringIndexer, VectorAssembler}
+import org.apache.spark.ml.linalg.Vectors
+import org.apache.spark.ml.parity.SparkParityBase
+import org.apache.spark.ml.{Pipeline, Transformer}
+import org.apache.spark.sql.DataFrame
+
+class LinearSVCParitySpec extends SparkParityBase
+{
+    override val dataset: DataFrame = baseDataset.select("fico_score_group_fnl", "dti")
+    override val sparkTransformer: Transformer = new Pipeline()
+      .setStages(Array(
+        new StringIndexer().
+            setInputCol("fico_score_group_fnl").
+            setOutputCol("fico_index"),
+        new VectorAssembler().
+                setInputCols(Array("fico_index", "dti")).
+                setOutputCol("features"),
+        new LinearSVCModel("linear_svc",
```

Comment:
can we not use the transformer LinearSVC?

Reply:
No we can't use the LinearSVC transformer, because if we use a transformer, then it will fit the model with their own coefficients and intercepts. But we want to have the model with our provided coefficients and intercept values. So we have to use the LinearSVCModel() method only.